sources/packages/backend/apps/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -181,10 +181,13 @@ export class StudentProfileAPIOutDTO {
   contact: ContactInformationAPIOutDTO;
   validSin: boolean;
   disabilityStatus: DisabilityStatus;
+}
+
+export class SensitiveDataStudentProfileAPIOutDTO extends StudentProfileAPIOutDTO {
```

Comment:
We probably have web dtos counterparts for these DTOs. Do they require any change also?

Reply:
I don't think we should change the web. In the web we are using the same DTO for all cases:
![image](https://github.com/bcgov/SIMS/assets/78114138/ae6e8e1d-1d30-419f-a442-f25f77cbce0f)
and in runtime JS doesn't complain about it: 

![image](https://github.com/bcgov/SIMS/assets/78114138/ef3e7ee5-96eb-453f-b9c6-8ee8ed7d685d)
Institution

![image](https://github.com/bcgov/SIMS/assets/78114138/5f939378-cd0c-4f36-889f-501d962be6bb)
Student

Reply:
But in any case, student profile DTO must not contain SIN right?

Reply:
then shouldn't be make `sin` as optional in web?

Reply:
I believe that the below one can reflect both possible DTOs received from the API.
https://github.com/bcgov/SIMS/blob/43ca26b8b71e1329c9c01819bbfbaf28ae48b9ed/sources/packages/web/src/services/http/StudentApi.ts#L46-L52

The service is already returning a `StudentProfile` that can have optional properties like SIN, as @ann-aot mentioned.

https://github.com/bcgov/SIMS/blob/43ca26b8b71e1329c9c01819bbfbaf28ae48b9ed/sources/packages/web/src/services/StudentService.ts#L52-L62

Does it make sense?